### ISSUE TITLE
Peer is disconnecting us because ... attempting to push a block that is too old

### ISSUE THREAD

**Issue by abitmore** [cryptonomex/graphene#262]
That's probably why it's hard to get into sync after got out of sync?

```
2015-08-22T22:20:33 p2p:message read_loop on_closing_connectio ] Peer 45.55.6.216:1776 is disconnecting us because of an error: You offered us a block that we reject as invalid, exception: {"code":10,"name":"assert_exception","message":"Assert Exception","stack":[{"context":{"level":"error","file":"fork_database.cpp","line":69,"method":"_push_block","hostname":"","thread_name":"th_a","timestamp":"2015-08-22T22:20:33"},"format":"item->num > std::max<int64_t>( 0, int64_t(_head->num) - (_max_size) ): attempting to push a block that is too old","data":{"item->num":174057,"head":174075,"max_size":10}},{"context":{"level":"warn","file":"db_block.cpp","line":176,"method":"_push_block","hostname":"","thread_name":"th_a","timestamp":"2015-08-22T22:20:33"},"format":"","data":{"new_block":{"previous":"0002a7e8b9613cd334a36aeccdb1d32460d8fa53","timestamp":"2015-08-22T22:20:14","witness":"1.6.4","next_secret_hash":"a6b9319cdc3465202c6ee6ba20f6fcfb4eb26211","previous_secret":"d879b9f9a141a1f9edc662fad77dcfa068e6562c","transaction_merkle_root":"0000000000000000000000000000000000000000","extensions":[],"witness_signature":"2055c39c86e1dd8acc632ea55d010ce56dd01601106c3367fa5ae6b3d08d35a5567a545a0a23a1bace2bf651c049a2559ba75f8ba87b0e597fe9ca88f113a2804e","transactions":[]}}},{"context":{"level":"warn","file":"application.cpp","line":379,"method":"handle_block","hostname":"","thread_name":"th_a","timestamp":"2015-08-22T22:20:33"},"format":"","data":{"blk_msg":{"block":{"previous":"0002a7e8b9613cd334a36aeccdb1d32460d8fa53","timestamp":"2015-08-22T22:20:14","witness":"1.6.4","next_secret_hash":"a6b9319cdc3465202c6ee6ba20f6fcfb4eb26211","previous_secret":"d879b9f9a141a1f9edc662fad77dcfa068e6562c","transaction_merkle_root":"0000000000000000000000000000000000000000","extensions":[],"witness_signature":"2055c39c86e1dd8acc632ea55d010ce56dd01601106c3367fa5ae6b3d08d35a5567a545a0a23a1bace2bf651c049a2559ba75f8ba87b0e597fe9ca88f113a2804e","transactions":[]},"block_id":"0002a7e934a552405ffc797448a046e0bbf916d9"},"sync_mode":true}}]}                     node.cpp:2666
```

**Comment by abitmore** [comment]
Related logs:

```
2015-08-22T22:20:31 th_a:Witness Block Production block_production_loo ] slot: 18 scheduled_witness: 1.6.69 scheduled_time: 2015-08-22T22:20:32 now: 2015-08-22T22:20:32                        witness.cpp:240
2015-08-22T22:20:32 th_a:invoke handle_block         handle_block ] Got block #174074 from network                      application.cpp:348
2015-08-22T22:20:32 th_a:invoke handle_block           push_block ] Pushing block to fork database that failed to link.
        fork_database.cpp:57
2015-08-22T22:20:32 th_a:invoke get_blockchain_synopsis get_blockchain_synop ] reference_point: 0000000000000000000000000000000000000000 number_of_blocks_after_reference_point: 0 result: ["0000a7e9879f3533922e34a2749fd0923213a23b","0001a7e9d307c5aa6cd8d2241eccbb236ca95df5","000227e941f966409ec1c773e3df0559e192361f","000267e948178758d3204457002db2494a351c0c","000287e9ec19f697eb11a18e5ef2df2ff18edb0c","000297e93de86fb634522eef5bce70d319594b6d","00029fe99aae786de2af1d97d889562dfafbf7a5","0002a3e923627564db1b4cb23a3c028a18437c3f","0002a5e9226b5e94953812e672237a35fd639054","0002a6e979268580885911bc471e5f8ee629f10b","0002a769000a78e75ab6c139f8dca26333a231de","0002a7a9286fc2ff9ac94e9f09a99e17f0f554ec","0002a7c90b260bf8bd6eb48c9aaf829810e6221e","0002a7d9856ad9e3213247d05e312b12142beca7","0002a7e1c883436be2682d0aa92dd77154e14c30","0002a7e5d8ced676183177d06aa0fff6607939b6","0002a7e7d0010c719d124b4c89ac2fd6e0f05349","0002a7e8b9613cd334a36aeccdb1d32460d8fa53","0002a7e934a552405ffc797448a046e0bbf916d9"]                   application.cpp:492
2015-08-22T22:20:32 th_a:Witness Block Production block_production_loo ] slot: 19 scheduled_witness: 1.6.58 scheduled_time: 2015-08-22T22:20:33 now: 2015-08-22T22:20:33                        witness.cpp:240
2015-08-22T22:20:33  th_a:invoke get_item             get_item ] Request for item {"item_type":1001,"item_hash":"0002a7e934a552405ffc797448a046e0bbf916d9"}                     application.cpp:443
2015-08-22T22:20:33  th_a:invoke get_item             get_item ] Serving up block #174057                       application.cpp:451
2015-08-22T22:20:33  th_a:invoke get_item             get_item ] Request for item {"item_type":1001,"item_hash":"0002a7e934a552405ffc797448a046e0bbf916d9"}                     application.cpp:443
2015-08-22T22:20:33  th_a:invoke get_item             get_item ] Serving up block #174057                       application.cpp:451
...
...
2015-08-22T22:20:33 th_a:Witness Block Production block_production_loo ] slot: 20 scheduled_witness: 1.6.75 scheduled_time: 2015-08-22T22:20:34 now: 2015-08-22T22:20:34                        witness.cpp:240
2015-08-22T22:20:34 th_a:invoke handle_block         handle_block ] Got block #174076 from network                      application.cpp:348
2015-08-22T22:20:34 th_a:invoke handle_block           push_block ] Pushing block to fork database that failed to link.
        fork_database.cpp:57
2015-08-22T22:20:34 th_a:invoke get_blockchain_synopsis get_blockchain_synop ] reference_point: 0000000000000000000000000000000000000000 number_of_blocks_after_reference_point: 0 result: ["0000a7e9879f3533922e34a2749fd0923213a23b","0001a7e9d307c5aa6cd8d2241eccbb236ca95df5","000227e941f966409ec1c773e3df0559e192361f","000267e948178758d3204457002db2494a351c0c","000287e9ec19f697eb11a18e5ef2df2ff18edb0c","000297e93de86fb634522eef5bce70d319594b6d","00029fe99aae786de2af1d97d889562dfafbf7a5","0002a3e923627564db1b4cb23a3c028a18437c3f","0002a5e9226b5e94953812e672237a35fd639054","0002a6e979268580885911bc471e5f8ee629f10b","0002a769000a78e75ab6c139f8dca26333a231de","0002a7a9286fc2ff9ac94e9f09a99e17f0f554ec","0002a7c90b260bf8bd6eb48c9aaf829810e6221e","0002a7d9856ad9e3213247d05e312b12142beca7","0002a7e1c883436be2682d0aa92dd77154e14c30","0002a7e5d8ced676183177d06aa0fff6607939b6","0002a7e7d0010c719d124b4c89ac2fd6e0f05349","0002a7e8b9613cd334a36aeccdb1d32460d8fa53","0002a7e934a552405ffc797448a046e0bbf916d9"]                   application.cpp:492
2015-08-22T22:20:34  th_a:invoke get_item             get_item ] Request for item {"item_type":1001,"item_hash":"0002a7e8b9613cd334a36aeccdb1d32460d8fa53"}                     application.cpp:443
2015-08-22T22:20:34  th_a:invoke get_item             get_item ] Serving up block #174056                       application.cpp:451
2015-08-22T22:20:34  th_a:invoke get_item             get_item ] Request for item {"item_type":1001,"item_hash":"0002a7e8b9613cd334a36aeccdb1d32460d8fa53"}                     application.cpp:443
2015-08-22T22:20:34  th_a:invoke get_item             get_item ] Serving up block #174056                       application.cpp:451
```

**Comment by abitmore** [comment]
Perhaps related p2p.log:

```
2015-08-22T22:20:32 p2p:message read_loop trigger_p2p_network_ ] Triggering connect loop now                    node.cpp:974
2015-08-22T22:20:32 p2p:message read_loop         send_message ] peer_connection::send_message() enqueueing message of type 5012 for p
eer 45.55.6.216:1776                      peer_connection.cpp:365
2015-08-22T22:20:32 p2p:message read_loop send_queueable_messa ] peer_connection::send_message() is firing up send_queued_message_task
                  peer_connection.cpp:354
2015-08-22T22:20:32 p2p:send_queued_messages_task              counter ] entering peer_connection::send_queued_messages_task()
                peer_connection.cpp:279
2015-08-22T22:20:32 p2p:send_queued_messages_task send_queued_messages ] peer_connection::send_queued_messages_task() calling message_
oriented_connection::send_message() to send message of type 5012 for peer 45.55.6.216:1776                        peer_connection.cpp:
291
2015-08-22T22:20:32 p2p:send_queued_messages_task send_queued_messages ] peer_connection::send_queued_messages_task()'s call to message_oriented_connection::send_message() completed normally for peer 45.55.6.216:1776                        peer_connection.cpp:294
2015-08-22T22:20:32 p2p:send_queued_messages_task send_queued_messages ] leaving peer_connection::send_queued_messages_task() due to queue exhaustion                   peer_connection.cpp:326
2015-08-22T22:20:32 p2p:send_queued_messages_task             ~counter ] leaving peer_connection::send_queued_messages_task()
                peer_connection.cpp:280
2015-08-22T22:20:32 p2p:message read_loop fetch_next_batch_of_ ] sync: sending a request for the next items after 0002a7e934a552405ffc
797448a046e0bbf916d9 to peer 45.55.6.216:1776, (full request is ["0000a7e9879f3533922e34a2749fd0923213a23b","0001a7e9d307c5aa6cd8d2241
eccbb236ca95df5","000227e941f966409ec1c773e3df0559e192361f","000267e948178758d3204457002db2494a351c0c","000287e9ec19f697eb11a18e5ef2df
2ff18edb0c","000297e93de86fb634522eef5bce70d319594b6d","00029fe99aae786de2af1d97d889562dfafbf7a5","0002a3e923627564db1b4cb23a3c028a184
37c3f","0002a5e9226b5e94953812e672237a35fd639054","0002a6e979268580885911bc471e5f8ee629f10b","0002a769000a78e75ab6c139f8dca26333a231de","0002a7a9286fc2ff9ac94e9f09a99e17f0f554ec","0002a7c90b260bf8bd6eb48c9aaf829810e6221e","0002a7d9856ad9e3213247d05e312b12142beca7","0002a7e1c883436be2682d0aa92dd77154e14c30","0002a7e5d8ced676183177d06aa0fff6607939b6","0002a7e7d0010c719d124b4c89ac2fd6e0f05349","0002a7e8b9613cd334a36aeccdb1d32460d8fa53","0002a7e934a552405ffc797448a046e0bbf916d9"])                       node.cpp:2290
2015-08-22T22:20:32 p2p:message read_loop         send_message ] peer_connection::send_message() enqueueing message of type 5003 for peer 45.55.6.216:1776                      peer_connection.cpp:365
2015-08-22T22:20:32 p2p:message read_loop send_queueable_messa ] peer_connection::send_message() is firing up send_queued_message_task                  peer_connection.cpp:354
2015-08-22T22:20:32 p2p:send_queued_messages_task              counter ] entering peer_connection::send_queued_messages_task()
                peer_connection.cpp:279
2015-08-22T22:20:32 p2p:send_queued_messages_task send_queued_messages ] peer_connection::send_queued_messages_task() calling message_oriented_connection::send_message() to send message of type 5003 for peer 45.55.6.216:1776                        peer_connection.cpp:291
2015-08-22T22:20:32 p2p:send_queued_messages_task send_queued_messages ] peer_connection::send_queued_messages_task()'s call to message_oriented_connection::send_message() completed normally for peer 45.55.6.216:1776                        peer_connection.cpp:294
2015-08-22T22:20:32 p2p:send_queued_messages_task send_queued_messages ] leaving peer_connection::send_queued_messages_task() due to queue exhaustion                   peer_connection.cpp:326
2015-08-22T22:20:32 p2p:send_queued_messages_task             ~counter ] leaving peer_connection::send_queued_messages_task()
                peer_connection.cpp:280
2015-08-22T22:20:32 p2p:message read_loop           on_message ] handling message current_time_request_message_type e5780688f0e9b1e00acb9ff50817d92f749c9a10 size 8 from peer 45.55.6.216:1776                  node.cpp:1651
2015-08-22T22:20:32 p2p:message read_loop         send_message ] peer_connection::send_message() enqueueing message of type 5013 for peer 45.55.6.216:1776                      peer_connection.cpp:365
2015-08-22T22:20:32 p2p:message read_loop send_queueable_messa ] peer_connection::send_message() is firing up send_queued_message_task                  peer_connection.cpp:354
2015-08-22T22:20:32 p2p:send_queued_messages_task              counter ] entering peer_connection::send_queued_messages_task()
                peer_connection.cpp:279
2015-08-22T22:20:32 p2p:send_queued_messages_task send_queued_messages ] peer_connection::send_queued_messages_task() calling message_oriented_connection::send_message() to send message of type 5013 for peer 45.55.6.216:1776                        peer_connection.cpp:291
2015-08-22T22:20:32 p2p:message read_loop           on_message ] handling message fetch_blockchain_item_ids_message_type 5075c67ce59f002f5435995134ef22a87e859ad3 size 385 from peer 45.55.6.216:1776                   node.cpp:1651
2015-08-22T22:20:32 p2p:message read_loop on_fetch_blockchain_ ] sync: received a request for item ids after {"item_type":1001,"item_hash":"0002a7e8b9613cd334a36aeccdb1d32460d8fa53"} from peer 45.55.6.216:1776 (full request: ["0000a7e847efe59b6e124af62687166acce0ebaf","0001a7e8ec8c070259f8d773d05750813e6038b2","000227e88efb25dc18d90992807df55854bb47c5","000267e8827c24922fe2db83a225bbaf30d4f14f","000287e88b33bbee02553262b732fcf0f57b24c7","000297e8b93a30947ec210bd69684375982a13ab","00029fe8a3b6d4f54831b463a111cf43c9bfaa24","0002a3e80aa70879302f358cd9375ae4f597f6aa","0002a5e827e1137726c59010170fd155bd40e4a7","0002a6e87a70bb57c3abb6002798d410d2c54836","0002a768b2eea4137e0fdfb9c67c2c6bc530b5e7","0002a7a84fdbc1df066c99e9608de8ccaf668f28","0002a7c841dbd9b2ca43332b659e0b8c59d85b64","0002a7d831c2e76a98e6915fc7b0801c14edaf21","0002a7e06f888216caba18a7ac32d829d069f174","0002a7e470fa4dc69231e31606933559d8b15735","0002a7e6e36d2ebacc713ef5918fc694a390c69e","0002a7e7d0010c719d124b4c89ac2fd6e0f05349","0002a7e8b9613cd334a36aeccdb1d32460d8fa53"])                    node.cpp:2132
2015-08-22T22:20:32 p2p:send_queued_messages_task send_queued_messages ] peer_connection::send_queued_messages_task()'s call to message_oriented_connection::send_message() completed normally for peer 45.55.6.216:1776                        peer_connection.cpp:294
2015-08-22T22:20:32 p2p:send_queued_messages_task send_queued_messages ] leaving peer_connection::send_queued_messages_task() due to queue exhaustion                   peer_connection.cpp:326
2015-08-22T22:20:32 p2p:send_queued_messages_task             ~counter ] leaving peer_connection::send_queued_messages_task()
                peer_connection.cpp:280
2015-08-22T22:20:32 p2p:message read_loop on_fetch_blockchain_ ] reply_message: {"total_remaining_item_count":0,"item_type":1001,"item_hashes_available":["0002a7e8b9613cd334a36aeccdb1d32460d8fa53","0002a7e934a552405ffc797448a046e0bbf916d9"]} fetch_blockchain_item_ids_message_received.blockchain_synopsis: ["0000a7e847efe59b6e124af62687166acce0ebaf","0001a7e8ec8c070259f8d773d05750813e6038b2","000227e88efb25dc18d90992807df55854bb47c5","000267e8827c24922fe2db83a225bbaf30d4f14f","000287e88b33bbee02553262b732fcf0f57b24c7","000297e8b93a30947ec210bd69684375982a13ab","00029fe8a3b6d4f54831b463a111cf43c9bfaa24","0002a3e80aa70879302f358cd9375ae4f597f6aa","0002a5e827e1137726c59010170fd155bd40e4a7","0002a6e87a70bb57c3abb6002798d410d2c54836","0002a768b2eea4137e0fdfb9c67c2c6bc530b5e7","0002a7a84fdbc1df066c99e9608de8ccaf668f28","0002a7c841dbd9b2ca43332b659e0b8c59d85b64","0002a7d831c2e76a98e6915fc7b0801c14edaf21","0002a7e06f888216caba18a7ac32d829d069f174","0002a7e470fa4dc69231e31606933559d8b15735","0002a7e6e36d2ebacc713ef5918fc694a390c69e","0002a7e7d0010c719d124b4c89ac2fd6e0f05349","0002a7e8b9613cd334a36aeccdb1d32460d8fa53"]                    node.cpp:2143
2015-08-22T22:20:32 p2p:message read_loop on_fetch_blockchain_ ] sync: peer is out of sync, sending peer 2 items ids: first: 0002a7e8b9613cd334a36aeccdb1d32460d8fa53, last: 0002a7e934a552405ffc797448a046e0bbf916d9                   node.cpp:2181
2015-08-22T22:20:32 p2p:message read_loop         send_message ] peer_connection::send_message() enqueueing message of type 5002 for peer 45.55.6.216:1776                      peer_connection.cpp:365
2015-08-22T22:20:32 p2p:message read_loop send_queueable_messa ] peer_connection::send_message() is firing up send_queued_message_task                  peer_connection.cpp:354
2015-08-22T22:20:32 p2p:send_queued_messages_task              counter ] entering peer_connection::send_queued_messages_task()
                peer_connection.cpp:279
2015-08-22T22:20:32 p2p:send_queued_messages_task send_queued_messages ] peer_connection::send_queued_messages_task() calling message_oriented_connection::send_message() to send message of type 5002 for peer 45.55.6.216:1776                        peer_connection.cpp:291
2015-08-22T22:20:32 p2p:send_queued_messages_task send_queued_messages ] peer_connection::send_queued_messages_task()'s call to message_oriented_connection::send_message() completed normally for peer 45.55.6.216:1776                        peer_connection.cpp:294
2015-08-22T22:20:32 p2p:send_queued_messages_task send_queued_messages ] leaving peer_connection::send_queued_messages_task() due to queue exhaustion                   peer_connection.cpp:326
2015-08-22T22:20:32 p2p:send_queued_messages_task             ~counter ] leaving peer_connection::send_queued_messages_task()
                peer_connection.cpp:280
2015-08-22T22:20:33 p2p:message read_loop           on_message ] handling message current_time_reply_message_type b8bfc872e69729b27a4c3095f43c9e7193ade860 size 24 from peer 45.55.6.216:1776                   node.cpp:1651
2015-08-22T22:20:33 p2p:message read_loop           on_message ] handling message blockchain_item_ids_inventory_message_type 31309d6cc8dfc0ff88a1b1bff827a740a4fff8ec size 29 from peer 45.55.6.216:1776                        node.cpp:1651
2015-08-22T22:20:33 p2p:message read_loop on_blockchain_item_i ] sync: received a list of 1 available items from 45.55.6.216:1776
                       node.cpp:2306
2015-08-22T22:20:33 p2p:message read_loop on_blockchain_item_i ] sync: peer said we're up-to-date, entering normal operation with this
 peer                     node.cpp:2321
...
...
2015-08-22T22:20:33 p2p:message read_loop           on_message ] handling message fetch_items_message_type 9774bfb6b6ee437cd1920b9740afa0592e2ddb37 size 25 from peer 45.55.6.216:1776                  node.cpp:1651
2015-08-22T22:20:33 p2p:message read_loop on_fetch_items_messa ] received items request for ids ["0002a7e934a552405ffc797448a046e0bbf916d9"] of type 1001 from peer 45.55.6.216:1776                    node.cpp:2508
2015-08-22T22:20:33 p2p:message read_loop on_fetch_items_messa ] received item request from peer 45.55.6.216:1776, returning the item from delegate with id 291ef0c0d0576452c14975696ce0cb1d11a96e97 size 172                   node.cpp:2538
2015-08-22T22:20:33 p2p:message read_loop            send_item ] peer_connection::send_item() enqueueing message of type 1001 for peer 45.55.6.216:1776                 peer_connection.cpp:374
2015-08-22T22:20:33 p2p:message read_loop send_queueable_messa ] peer_connection::send_message() is firing up send_queued_message_task                  peer_connection.cpp:354
2015-08-22T22:20:33 p2p:send_queued_messages_task              counter ] entering peer_connection::send_queued_messages_task()
                peer_connection.cpp:279
2015-08-22T22:20:33 p2p:send_queued_messages_task send_queued_messages ] peer_connection::send_queued_messages_task() calling message_oriented_connection::send_message() to send message of type 1001 for peer 45.55.6.216:1776                        peer_connection.cpp:291
2015-08-22T22:20:33 p2p:send_queued_messages_task send_queued_messages ] peer_connection::send_queued_messages_task()'s call to message_oriented_connection::send_message() completed normally for peer 45.55.6.216:1776                        peer_connection.cpp:294
2015-08-22T22:20:33 p2p:send_queued_messages_task send_queued_messages ] leaving peer_connection::send_queued_messages_task() due to queue exhaustion                   peer_connection.cpp:326
2015-08-22T22:20:33 p2p:send_queued_messages_task             ~counter ] leaving peer_connection::send_queued_messages_task()
                peer_connection.cpp:280
...
...
2015-08-22T22:20:33 p2p:message read_loop           on_message ] handling message closing_connection_message_type 548113417bb6f43926946b81bc100840e0b33f3f size 1738 from peer 45.55.6.216:1776                 node.cpp:1651
2015-08-22T22:20:33 p2p:message read_loop on_closing_connectio ] Peer 45.55.6.216:1776 is disconnecting us because of an error: You offered us a block that we reject as invalid, exception: {"code":10,"name":"assert_exception","message":"Assert Exception","stack":[{"context":{"level":"error","file":"fork_database.cpp","line":69,"method":"_push_block","hostname":"","thread_name":"th_a","timestamp":"2015-08-22T22:20:33"},"format":"item->num > std::max<int64_t>( 0, int64_t(_head->num) - (_max_size) ): attempting to push a block that is too old","data":{"item->num":174057,"head":174075,"max_size":10}},{"context":{"level":"warn","file":"db_block.cpp","line":176,"method":"_push_block","hostname":"","thread_name":"th_a","timestamp":"2015-08-22T22:20:33"},"format":"","data":{"new_block":{"previous":"0002a7e8b9613cd334a36aeccdb1d32460d8fa53","timestamp":"2015-08-22T22:20:14","witness":"1.6.4","next_secret_hash":"a6b9319cdc3465202c6ee6ba20f6fcfb4eb26211","previous_secret":"d879b9f9a141a1f9edc662fad77dcfa068e6562c","transaction_merkle_root":"0000000000000000000000000000000000000000","extensions":[],"witness_signature":"2055c39c86e1dd8acc632ea55d010ce56dd01601106c3367fa5ae6b3d08d35a5567a545a0a23a1bace2bf651c049a2559ba75f8ba87b0e597fe9ca88f113a2804e","transactions":[]}}},{"context":{"level":"warn","file":"application.cpp","line":379,"method":"handle_block","hostname":"","thread_name":"th_a","timestamp":"2015-08-22T22:20:33"},"format":"","data":{"blk_msg":{"block":{"previous":"0002a7e8b9613cd334a36aeccdb1d32460d8fa53","timestamp":"2015-08-22T22:20:14","witness":"1.6.4","next_secret_hash":"a6b9319cdc3465202c6ee6ba20f6fcfb4eb26211","previous_secret":"d879b9f9a141a1f9edc662fad77dcfa068e6562c","transaction_merkle_root":"0000000000000000000000000000000000000000","extensions":[],"witness_signature":"2055c39c86e1dd8acc632ea55d010ce56dd01601106c3367fa5ae6b3d08d35a5567a545a0a23a1bace2bf651c049a2559ba75f8ba87b0e597fe9ca88f113a2804e","transactions":[]},"block_id":"0002a7e934a552405ffc797448a046e0bbf916d9"},"sync_mode":true}}]}                    node.cpp:2666
```

**Comment by vikramrajkumar** [comment]
This issue was moved to bitshares/bitshares-core#72
